server/src/main/java/no/fractal/socket/send/AbstractMessage.java

```diff
@@ -73,21 +86,19 @@ protected Segment(Map<String, String> segmentMeta) {
             return segmentMeta;
         }
 
-        public int getBodyByteSize(){
+        public int getBodyByteSize() {
             return Integer.parseInt(segmentMeta.get(keys.size.name()));
         }
 
         public abstract void writeToStream(BufferedOutputStream outputStream) throws IOException;
 
-        protected enum keys{
-            size,
-            mime_type,
-            file_name
+        protected enum keys {
+            size, mime_type, file_name
```

Comment:
nei nei nei det her e jo en helt syk måte å skriv enum's på.

Reply:
Kan ikke noe for det :D sorrrry, haha